Sources/Core/AWSClientRuntime/Auth/Credentials.swift

```diff
@@ -6,34 +6,35 @@
 //
 
 import Foundation
+import ClientRuntime
```

Comment:
Try `import protocol ClientRuntime.Identity` here.

Making imports more specific helps to expose what the dependencies of a single source file are.